dubbo-metadata/dubbo-metadata-report-redis/src/main/java/org/apache/dubbo/metadata/store/redis/RedisMetadataReport.java

```diff
@@ -207,4 +226,261 @@ private String getMetadataStandalone(BaseMetadataIdentifier metadataIdentifier)
         }
     }
 
+    @Override
+    public boolean registerServiceAppMapping(String serviceInterface, String defaultMappingGroup, String newConfigContent, Object ticket) {
+        try {
+            if (null!= ticket && !(ticket instanceof String)) {
+                throw new IllegalArgumentException("zookeeper publishConfigCas requires stat type ticket");
```

Comment:
Should be redis here

Reply:
Okay, I will fix it in this link: https://github.com/apache/dubbo/pull/13303. Should I close this PR or that one?

Reply:
Close this one and track in #13303 in the future